diff --git a/config.def.h b/config.def.h
index d805331..0923187 100644
--- a/config.def.h
+++ b/config.def.h
@@ -65,5 +65,10 @@ static const char unknown_str[] = "n/a";
  */
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	{ username, " [%s]",           NULL },
+	{ run_command, " [󱄠 %s]",           "wpctl get-volume @DEFAULT_AUDIO_SINK@ | sed 's/Volume: //'" },
+	{ disk_free, " [󰋊 %s]",           "/" },
+	{ cpu_perc, " [󰻠 %s\%]",           NULL },
+	{ ram_perc, " [󰍛: %s\%]",           NULL },
+	{ datetime, " [%s]",           "%d/%m/%Y %H:%M:%S" },
 };
